src/app/trainees/trainees.component.html

```diff
@@ -1,8 +1,8 @@
 <app-trainee-filters></app-trainee-filters>
 
-<div class="d-grid trainees-top-section align-items-center">
+<div class="d-grid trainees-top-section align-items-center sticky-toolbar">
```

Comment:
maybe desktop only? takes up real-estate on mobile.
![Screenshot 2020-05-18 at 13 43 43](https://user-images.githubusercontent.com/13540819/82214369-a16b2680-990d-11ea-9739-c5a151f36cbf.png)

Reply:
i think its a good idea for the sticky content to be present on both mobile and desktop. the padding can be tweaked for mobile view to make it more tighter but this looks good

Reply:
yeah we'll look at padding across the board. maincontent padding for mobile, pad, desktop to have same as toolbar and then max-width for the whole UI container at some point.